ci/build-wheel-deps.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
 set -e -x
 
-PACKAGES=(
-    zlib-1.2.11
-    libpng-1.6.37
-    freetype-2.10.4
-    harfbuzz-2.6.4
-)
+curl -L "https://downloads.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.gz" -o libpng-1.6.37.tar.gz
+tar zxf libpng-1.6.37.tar.gz
+pushd libpng-1.6.37
+./configure
+make -j $(nproc)
+make install
+popd
 
-URLS=(
-    https://www.zlib.net/zlib-1.2.11.tar.gz
-    https://downloads.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.gz
-    https://download.savannah.gnu.org/releases/freetype/freetype-2.10.4.tar.gz
-    https://github.com/harfbuzz/harfbuzz/archive/2.6.4.tar.gz
-)
+curl -L "https://download.savannah.gnu.org/releases/freetype/freetype-2.12.1.tar.gz" -o freetype-2.12.1.tar.gz
+tar zxf freetype-2.12.1.tar.gz
+pushd freetype-2.12.1
+./configure
+make -j $(nproc)
+make install
+popd
```

Comment:
Yeah. Sometimes doing the simplest thing is the best. DRY was not helping before 😄